app/views/candidate_mailer/find_another_course.text.erb

```diff
@@ -1,19 +0,0 @@
-Dear <%= @application_form.first_name %>
```

Comment:
Why is this deleted @stevehook? Is it because we currently don't call it anywhere?

I seem to remember I drafted a ticket up about this - we probably should be calling it and letting candidates now.

What are your thoughts?